adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -473,6 +507,25 @@ def connect_AP(self, ssid, password): # pylint: disable=invalid-name
             raise RuntimeError("No such ssid", ssid)
         raise RuntimeError("Unknown error 0x%02X" % stat)
 
+    def create_ap(self, ssid, password, channel=b'\x01'):
+        """Create an access point with the given name and password."""
```

Comment:
Document the kwargs for `create_ap` here.

Reply:
will document all of the args 👍